sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -64,11 +67,33 @@ def enable_capture_replay(self, value):
     and pipeline runs always use the same data captured; False - Disables
     capture of replayable source data so that following PCollection evaluation
     and pipeline runs always use new data from sources."""
+    # This makes sure the log handler is configured correctly in case the
+    # options are configured in an early stage.
+    _ = ie.current_env()
+    if value:
+      _LOGGER.info(
+          'Capture replay is enabled. When a PCollection is evaluated or the '
+          'pipeline is executed, existing data captured from previous '
+          'computations will be replayed for replayability. If no capture is '
```

Comment:
Replayed for replayability? How about this? I'm not sure what's a good phrasing, but... 

Also, maybe capture->captured data?
```suggestion
          'computations will be replayed for consistent results. If no captured data is '
```